akka-cluster-sharding-typed/src/test/java/jdocs/akka/cluster/sharding/typed/HelloWorldPersistentEntityExample.java

```diff
@@ -43,7 +43,7 @@ public HelloWorldService(ActorSystem<?> system) {
       sharding = ClusterSharding.get(system);
 
       sharding.init(
-          Entity.ofPersistentEntity(
+          Entity.ofEventSourcedEntity(
```

Comment:
Mention this rename in akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md
We will be more strict with API changes from now.

Reply:
ok, I will do it